src/main/pty/AgentDetector.ts

```
@@ -377,6 +384,61 @@ const MAX_BUFFER = 16 * 1024;
+
+function stripAnsi(line: string): string {
+  return line.includes('\u001b') ? line.replace(ANSI_STRIP, '') : line;
+}
+
+function visibleChrome(line: string): string {
+  return stripAnsi(line).replace(CHROME_NOISE_RE, ' ').replace(/\s+/g, ' ').trim();
+}
+
+/** OSC title, or a line whose visible text *starts* with `Claude Code`. */
+function isClaudeBannerChrome(line: string): boolean {
+  if (/\x1b\]0;[^\x07]*(?<!Open)(?<!Open\s)Claude/.test(line)) return true;
```

ESLint

[error] Unexpected control character(s) in regular expression: \x1b, \x07.

(no-control-regex)